boa_engine/src/builtins/array/mod.rs

```diff
@@ -2855,4 +2862,44 @@ impl Array {
             .constructor(false)
             .build()
     }
+
+    pub(crate) fn unscopables_intrinsic(context: &mut Context) -> JsObject {
+        let object_prototype = context.construct_object();
+        let result_obj = JsObject::from_proto_and_data(object_prototype, ObjectData::ordinary());
```

Comment:
```suggestion
        let result_obj = JsObject::empty();
```

The spec specifies that the created object must have a `null` prototype. This would be the shortest way to create it, but it can also be created with `JsObject::from_proto_and_data(None, ObjectData::ordinary())`, which is a bit more explicit. Choose the one you prefer :)

Reply:
Done. https://github.com/boa-dev/boa/pull/1963/files#diff-2a7aca2556bbe4b446193320bd39075e9013557432aa5f3c5832173098d7bc86R2880